code/game/machinery/computer/access_terminal.dm

```diff
@@ -0,0 +1,103 @@
+
+/// Simple access terminal.
+/// Can put an ID in it, and set or unset some accesses.
+/// Intended to be used in ooc/admin spaces, like antag/actor prep areas.
+ABSTRACT_TYPE(/obj/machinery/computer/access_terminal)
+	name = "self-service access terminal"
+	desc = "A simple access terminal. It allows changing one's ID accesses."
+	icon = 'icons/obj/computer.dmi'
+	icon_state = "altcomputerw"
+	light_color = LIGHT_COLOR_BLUE
+	icon_screen = "guest"
+	icon_scanline = "altcomputerw-scanline"
+	density = FALSE
+	appearance_flags = TILE_BOUND
+
+	var/obj/item/card/id/held_card
```

Comment:
https://github.com/Aurorastation/Aurora.3/blob/master/.github/guides/STYLE.md#what-to-document

`All variables, procs and classes that do not have a clear name / signature must be documented using DMDoc.`

Reply:
it's pretty obvious what card this var is referring to but ok